### PR TITLE
Child details page

### PR DESCRIPTION
**Pull request by mikolajmirko**
Additionally:
- _AppLoader_ widget with Fokus flower animation
- _AppHero_ widget for "no elements in the list" info and future use
- _AppAlert_ widget for important information box, currently use only on this page
- Changes to _AppHeader_ to include tabs and _ItemCard_ to be able to display icons instead of graphics

### CODE REVIEW

**Comment by mikolajmirko** [comment]
I updated the AppPage enum listing - I don't want to resolve conflicts in here for 10th time ;_;
code

**Comment by mikolajmirko** [comment]
Updated merge request description with summary.

File: i18n/en_US.json [hunk 1]
```diff
@@ -87,6 +87,37 @@
...
+					}
+				},
+				"content": {
+					"plansTitle": "Today's plans",
+					"noPlansText": "No actions planned for today",
+					"openCalendarButton": "Open calendar",
+					"awardsTitle": "Claimed awards",
+					"noAwardsText": "No claimed awards to display",
+					"achievementsTitle": "Awarded badges",
+					"noachievementsText": "No awarded badges to display",
```
Comment (stasgora) [inline i18n/en_US.json:112]: Typo, the 'a' should be capital

File: i18n/en_US.json [hunk 2]
```diff
@@ -87,6 +87,37 @@
...
+					"alerts": {
+						"noPlansAdded": "No plan added yet. Add a plan to be able to assign it to a child.",
+						"unratedTasksExist": "There are unrated tasks. Visit the rating page to assign points for completed tasks.",
+						"noAwardsAdded": "The list of awards available for purchase is empty. Add an award for a child to earn.",
```
Comment (stasgora) [inline i18n/en_US.json:116]: for a child to **claim**, maybe?

File: lib/pages/caregiver/child_dashboard_page.dart [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+      new _CaregiverChildDashboardPageState();
+}
+
+class _CaregiverChildDashboardPageState extends State<CaregiverChildDashboardPage> with TickerProviderStateMixin {
+	static const String _pageKey = 'page.caregiverSection.childDashboard';
+	TabController _tabController;
+	int _currentIndex = 0;
+
+	double customBottomBarHeight = 40.0;
+	Duration bottomBarAnimationDuration = Duration(milliseconds: 400);
```
Comment (stasgora) [inline lib/pages/caregiver/child_dashboard_page.dart:34]: Use static const / final for constants

**Review by stasgora — CHANGES_REQUESTED** [review]

**Comment by stasgora** [comment]
The PointPickerField widget still uses the CircularProgressIndicator ;)

**Comment by mikolajmirko** [comment]
I left it there because it's imitates the look of loading SmartSelect. It's grey and small - I tested it and Fokus flower looks not so good in this place (beside the fact I would need to update every SmartSelect with custom builder).

**Review by stasgora — APPROVED** [review]

**Pull request merged**